src/UserService.Models.Db/DbUser.cs

```diff
@@ -20,8 +24,37 @@ public class DbUser
         [Required]
```

Comment:
We use Fluent API, no need to use attributes

Reply:
done